### PR TITLE
Added Release Notes 6.5.0

### PR DESCRIPTION
**Pull request by tbsbdr**
- Public and Private Links in the Sharing Panel
- Copy “Permanent Link” by default
- Public Link: Copy with or without password to clipboard
- Removed link types: “Can upload” and “Invited people”
- “Save as” in the app top bar
- Right Sidebar Transition
- Version Information In ~~Right~~ Left Sidebar
- Build your own Web Application: Web App Boilerplate

### CODE REVIEW

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 1]
```diff
@@ -16,6 +16,55 @@
...
+== Infinite Scale 6.5.0 (Rolling)
+
+IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.
+
+[discrete]
+=== Public and Private Links in the Sharing Panel
+
+The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.
+
+- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it ex. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:28]:
```suggestion
- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved.
```

Comment (kulmann) [reply modules/ROOT/pages/ocis_release_notes.adoc:28]: Could you also add something like `The permanent link is located by the space memberships and people shares.`

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -16,6 +16,55 @@
...
+
+A “Save As” option is now available in the app top bar for ex. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.
+
+[discrete]
+=== Right Sidebar Transition
+
+The right sidebar transition has been optimized for smoother and more natural animation, enhancing the overall user experience.
+
+[discrete]
+=== Version Information In Right Sidebar
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:59]:
```suggestion
=== Version Information In Left Sidebar
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 3]
```diff
@@ -16,6 +16,55 @@
...
+
+The version information for Infinite Scale and the Web UI is now displayed in the bottom left sidebar. This enhancement ensures that users can easily identify the versions they are using at any time, simplifying the process for support requests.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:61]:
```suggestion
The version information for Infinite Scale and the Web UI is now displayed in the bottom of the left sidebar. This enhancement ensures that users can easily identify the software versions they are using at any time, simplifying the process for support requests.
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 4]
```diff
@@ -16,6 +16,55 @@
...
+
+[discrete]
+=== Removed link types: "Can upload" and "Invited people"
+
+The “Can upload” and “Invited people” link types have been removed to simplify sharing options. The functionality of the “Invited people” link is still available through the “Permanent link” option.
+
+[discrete]
+=== "Save as" in the app top bar
+
+A “Save As” option is now available in the app top bar for ex. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:51]:
```suggestion
A “Save As” option is now available in the app top bar for e.g. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 5]
```diff
@@ -16,6 +16,55 @@
...
+
+[discrete]
+=== Public Link: Copy with or without password to clipboard  
+
+You can now copy a public link to your clipboard with or without a password, giving you more control over file sharing.
+
+[discrete]
+=== Removed link types: "Can upload" and "Invited people"
+
+The “Can upload” and “Invited people” link types have been removed to simplify sharing options. The functionality of the “Invited people” link is still available through the “Permanent link” option.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:46]: Please add something like `Existing links of those types will continue to work. It is not possible to create new public links with these types though.`

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 6]
```diff
@@ -16,6 +16,55 @@
 
 toc::[]
 
+== Infinite Scale 6.5.0 (Rolling)
+
+IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.
+
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:22]:
```suggestion

Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.5.0[GitHub, window=_blank].
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 7]
```diff
@@ -16,6 +16,55 @@
...
+[discrete]
+=== Public and Private Links in the Sharing Panel
+
+The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.
+
+- *Permanent Link*: A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved. The permanent link is located by the space memberships and people shares.
+
+- *Public Links*: These links grant access to external people outside your organization. Public links are password-protected by default, ensuring secure file sharing with non-members. Password protection can be turned off by the system administrator.
+
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:31]:
```suggestion
- *Permanent Link*: +
A fixed link for internal use that works as a pointer for people with existing access. Use it e.g. to remind someone to review a file. The link remains unchanged, even if the file or folder is renamed or moved. The permanent link is located by the space memberships and people shares.

- *Public Links*: +
These links grant access to external people outside your organization. Public links are password-protected by default, ensuring secure file sharing with non-members. Password protection can be turned off by the system administrator.
```
Improves readbility

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 8]
```diff
@@ -16,6 +16,55 @@
...
+
+[discrete]
+=== Copy "Permanent Link" by default
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:34]:
```suggestion
=== Copy "Permanent Link" by Default
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 9]
```diff
@@ -16,6 +16,55 @@
...
+
+Copying a link in the file list now provides the permanent link with a single click. Permanent links work only for people with existing access, ensuring access is secure by default while offering a faster way to share and point people to files.
+
+[discrete]
+=== Public Link: Copy with or without password to clipboard
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:39]:
```suggestion
=== Public Link: Copy With or Without Password to Clipboard  
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 10]
```diff
@@ -16,6 +16,55 @@
...
+
+Copying a link in the file list now provides the permanent link with a single click. Permanent links work only for people with existing access, ensuring access is secure by default while offering a faster way to share and point people to files.
+
+[discrete]
+=== Public Link: Copy with or without password to clipboard  
+
+You can now copy a public link to your clipboard with or without a password, giving you more control over file sharing.
+
+[discrete]
+=== Removed link types: "Can upload" and "Invited people"
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:44]:
```suggestion
=== Removed Link Types: "Can upload" and "Invited people"
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 11]
```diff
@@ -16,6 +16,55 @@
...
+
+A “Save As” option is now available in the app top bar for e.g. markdown or plaintext files. You can easily save your document to the desired location without leaving the editor.
+
+[discrete]
+=== Right Sidebar Transition
+
+The right sidebar transition has been optimized for smoother and more natural animation, enhancing the overall user experience.
+
+[discrete]
+=== Version Information In Left Sidebar
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:59]:
```suggestion
=== Version Information in Left Sidebar
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 12]
```diff
@@ -16,6 +16,55 @@
...
+
+The version information for Infinite Scale and the Web UI is now displayed in the bottom of the left sidebar. This enhancement ensures that users can easily identify the software versions they are using at any time, simplifying the process for support requests.
+
+[discrete]
+=== Build your own Web Application: Web App Boilerplate
```
Comment (mmattel) [inline modules/ROOT/pages/ocis_release_notes.adoc:64]:
```suggestion
=== Build Your own Web Application: Web App Boilerplate
```

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 13]
```diff
@@ -16,6 +16,58 @@
...
 toc::[]
 
+== Infinite Scale 6.5.0 (Rolling)
+
+IMPORTANT: This is a Rolling Release. Please check the {release-types-url}[documentation] to see if this release type is right for your use case.
+
+Refer to the source and the full change log for a list of bug fixes and changes at {ocis-releases-url}/v6.5.0[GitHub, window=_blank].
+
+[discrete]
+=== Public and Private Links in the Sharing Panel
```
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:26]: You call them Public and Private in the heading, but the text uses Permanent and Public.

Edit the heading, or add some explanation that Private = Permanent

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 14]
```diff
@@ -16,6 +16,58 @@
...
+
+The sharing panel has been redesigned to separate links into two categories: Permanent Link and Public Links.
```
Comment (phil-davis) [inline modules/ROOT/pages/ocis_release_notes.adoc:28]:
```suggestion
The sharing panel has been redesigned to separate links into two categories: Permanent Links and Public Links.
```

**Review by micbar — APPROVED** [review]

**Review by phil-davis — COMMENTED** [review]
I was in the process of reviewing when this was merged. Just small comments @mmattel

**Comment by mmattel** [comment]
@phil-davis I will do a post fix with the changes you suggested.

**Pull request merged**